Assets/_Application/Layers/LayerTypes/Credentials/Properties/LayerCredentialsHandler.cs

```diff
@@ -50,7 +52,16 @@ private void OnEnable()
             layerWithCredentials.OnURLChanged.AddListener(UrlHasChanged);
             layerWithCredentials.OnServerResponseReceived.AddListener(HandleServerResponse);
 
-            UrlHasChanged(layerWithCredentials.URL);
+            UpdateUrl(layerWithCredentials.URL);
+        }
+
+        public void UpdateUrl(string url)
+        {
+            if (!string.IsNullOrEmpty(url))
```

Comment:
Moet dit niet een error / exception gooien als de url null of empty is? Want nu kan de applicatie stilletjes doorgaan en debuggen we ons suf waarom iets niet werkt :) En ook voor de gebruiker is het duidelijker als er iets mis gaat